### PR TITLE
Add link/include paths to examples

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: example/fly_mission/CMakeLists.txt [hunk 1]
```diff
@@ -4,12 +4,17 @@ project(fly_mission)
 
 if(NOT MSVC)
     add_definitions("-std=c++11 -Wall -Wextra -Werror")
+	# Line below required if /usr/local/include is not in your default includes
```
Comment (julianoes) [inline example/fly_mission/CMakeLists.txt]: Spaces not tabs please :smile:

File: example/fly_qgc_mission/CMakeLists.txt [hunk 2]
```diff
@@ -4,6 +4,10 @@ project(fly_qgc_mission)
 
 if(NOT MSVC)
     add_definitions("-std=c++11 -Wall -Wextra -Werror")
+	# Line below required if /usr/local/include is not in your default includes
```
Comment (julianoes) [inline example/fly_qgc_mission/CMakeLists.txt]: Same.

File: example/follow_me/CMakeLists.txt [hunk 3]
```diff
@@ -2,16 +2,26 @@ cmake_minimum_required(VERSION 2.8.12)
...
     include_directories(${CMAKE_SOURCE_DIR}/../../install/include)
     link_directories(${CMAKE_SOURCE_DIR}/../../install/lib)
     add_definitions("-std=c++11 -WX -W2")
+    find_package(Boost 1.64 COMPONENTS system date_time REQUIRED )
     # This is to avoid auto-linking of libraries that we don't build
     add_definitions("-DBOOST_ALL_NO_LIB")
+endif()
+
+if (MSVC)
+
```
Comment (julianoes) [inline example/follow_me/CMakeLists.txt]: What's this empty `if` `else`?

File: example/follow_me/CMakeLists.txt [hunk 4]
```diff
@@ -2,16 +2,26 @@ cmake_minimum_required(VERSION 2.8.12)
...
+	# Line below required if /usr/local/include is not in your default includes
+    #include_directories(/usr/local/include)
+    # Line below required if /usr/local/lib is not in your default linker path
+    #link_directories(/usr/local/lib)
+    find_package(Boost 1.58 COMPONENTS system REQUIRED )
+else()
     include_directories(${CMAKE_SOURCE_DIR}/../../install/include)
     link_directories(${CMAKE_SOURCE_DIR}/../../install/lib)
     add_definitions("-std=c++11 -WX -W2")
+    find_package(Boost 1.64 COMPONENTS system date_time REQUIRED )
```
Comment (julianoes) [inline example/follow_me/CMakeLists.txt]: `date_time` required, really?

Comment (hamishwillee) [reply example/follow_me/CMakeLists.txt]: You'll have to ask @shakthi-prashanth-m - I moved things around for consistency, but this was already here.

**Comment by hamishwillee** [comment]
Thanks @julianoes . All fixed except the question about that boost library. Arguably you could merge anyway though, since this is not a change.

File: example/follow_me/CMakeLists.txt (hunk 4 above)
Comment (julianoes) [reply example/follow_me/CMakeLists.txt]: Oh, you're right.

**Pull request merged**